### PR TITLE
saving and loading ability

### PR DESCRIPTION
**Pull request by guruprerana**

### CODE REVIEW

**Comment by ngthanhvinh** [comment]
Have you checked the case at which we create a portfolio with the same as a portfolio created before?

**Comment by guruprerana** [comment]
> Have you checked the case at which we create a portfolio with the same as a portfolio created before?

Right now, by design you are allowed to have two portfolios with the same name (ID) with the disadvantage that you would not be able to distinguish between them while choosing which portfolio you want to work with on startup.

Of course we could perform a check and show an error if there already exists a portfolio with the same name but this would require some more work and since I figured that one user would not create too many portfolios, this would not be a problem.

However, right now there is no issue with having two portofolios of same name since they are stored in an ordered vector and saved to the json in the same way. 

**TL;DR: Right now, I think it is OK to allow two portfolios of the same name, and it works with the current structure.**

**Review by ngthanhvinh — APPROVED** [review]
